utils/run_benchmarks.py

```diff
@@ -1,34 +1,46 @@
 import argparse
+import logging
 import os
+import re
+import shutil
+import subprocess
+import sys
+import time
 from collections import namedtuple
 from concurrent.futures import ProcessPoolExecutor
 from functools import partial
-import git
+from logging.handlers import RotatingFileHandler
+from logging import StreamHandler
 from os import path, symlink
-import pandas as pd
 from re import match
-import shutil
-import subprocess
-import sys
-from tabulate import tabulate
-import time
 from typing import Any, Tuple
+
+import git
+import pandas as pd
 import yaml
+from tabulate import tabulate
 
+from xl2times import utils
 from xl2times.utils import max_workers
 
+from loguru import logger
+
+logger = utils.get_logger()
 
-def parse_result(lastline):
-    m = match(
-        r"(\d+\.\d)\% of ground truth rows present in output \((\d+)/(\d+)\)"
-        r", (\d+) additional rows",
-        lastline,
+
+def parse_result(output: str) -> Tuple[float, int, int]:
+    # find pattern in multiline string
+    m = re.findall(
+        r"(\d+\.\d)% of ground truth rows present in output \((\d+)/(\d+)\), (\d+) additional rows",
+        output,
+        flags=re.MULTILINE,
```

Comment:
Curious about this -- does loguru split output into lines based on terminal width?

Reply:
Sorry, forgot to explain this one.  The old check wasn't matching anything for me because stdout had some control chars on the last line of the suffix.  I changed this function (and where it gets called) to just parse the entire stdout contents for this line.  So the re.MULTILINE just tells the regex call that it's matching a block of text with lots of newline chars.